### ISSUE TITLE
Attribute "layout" has already been defined

### ISSUE THREAD

**Issue by ghost** [alexandrius/accordion-swipe-layout#3]
The attribute "[layout](https://github.com/alexandrius/accordion-swipe-layout/blob/master/library/src/main/res/values/attrs.xml#L15)" clashes with the attribute "layout" of the android support library.
Could it be renamed to something more specific like foregroundLayout, baseItemLayout...

**Comment by alexandrius** [comment]
Great idea. Will change it

**Comment by alexandrius** [comment]
Until I update the library you can add custom layout as first child of the SwipeLayout. 

```java
<com.alexandrius.accordionswipelayout.library.SwipeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/swipe_layout"
    android:layout_width="match_parent"
    android:layout_height="80dp"
    app:iconSize="@dimen/icon_size"
    app:leftItemColors="@array/leftColors"
    app:leftItemIcons="@array/leftDrawables"
    app:rightItemColors="@array/rightColors"
    app:rightIconColors="@array/rightDrawableColors"
    app:canFullSwipeFromLeft="true"
    app:rightItemIcons="@array/rightDrawables"
    app:rightStrings="@array/rightTexts"
    app:rightTextColors="@array/rightTextColors"
    app:swipeItemWidth="@dimen/swipe_item_width"
    app:textSize="@dimen/text_size">

    <include layout="@layout/sample_item"/>
</com.alexandrius.accordionswipelayout.library.SwipeLayout>
```

**Comment by alexandrius** [comment]
Implemented in latest 0.5.0 version
